### PR TITLE
ui version2 - home page

### PR DESCRIPTION
**Pull request by arushigarg18**

### CODE REVIEW

File: ui/ui_v1/scripts/code.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//function to import css and javascript files into the html file
+function include(filename) {
+  return HtmlService.createHtmlOutputFromFile(filename).getContent();
+}
+
+//function to select the activated range from sheet
+function getSelectedRange() {
+  var range = SpreadsheetApp.getActiveSheet().getActiveRange(); 
+  var rangeString = range.getA1Notation(); 
+  var headerRow = range.getLastRow() - range.getHeight()+1;
```
Comment (SowmithManepalli) [inline ui/ui_v1/scripts/code.js]: This is implying that header row is always the row above the selection right ? If yes can you mention it here.

File: ui/ui_v1/scripts/process_query.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
+//json object that contains all aspects and table's data
+var JsonObj = {
```
Comment (SowmithManepalli) [inline ui/ui_v1/scripts/process_query.js]: use camel case here in the variable name.

File: ui/ui_v1/scripts/process_query.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+   * JsonObj.isAsc is a boolean variable which stores true for ascending order
+   */
+  JsonObj.isAsc = isAsc;
+  
+  /* 
+   * set limit
+   * JsonObj.k is an Integer storing K 
+   * (if user doesn't specify values of k -5000000 default)
+   */
+  JsonObj.k = k;
```
Comment (SowmithManepalli) [inline ui/ui_v1/scripts/process_query.js]: Modify variable name to something more obvious, i m assuming this is the k in top-k

File: ui/ui_v1/scripts/process_query.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+   
+  /*
+   * set slice
```
Comment (SowmithManepalli) [inline ui/ui_v1/scripts/process_query.js]: Explain what is a slice here, or link to a doc or something that has an explanation.

File: ui/ui_v1/scripts/process_query.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+   */
+  JsonObj.rowRange = objRowRange;
+
+  //sending entire sheet data 
+  var rangeAll = sheet.getDataRange();
+  return rangeAll.getValues();
+}
+
+//function to call gcp function to obtain result from the received data
+function callGcpFunc() {
```
Comment (SowmithManepalli) [inline ui/ui_v1/scripts/process_query.js]: Modify the function name , the name should imply what the gcp function is accomplishing.

File: ui/ui_v1/scripts/code.js (hunk 1 above)
Comment (arushigarg18) [reply ui/ui_v1/scripts/code.js]: It implies that header row will be the first row of the selected range. I will add this in comments here.

File: ui/ui_v1/scripts/process_query.js (hunk 3 above)
Comment (arushigarg18) [reply ui/ui_v1/scripts/process_query.js]: Will limit be fine ?

**Pull request merged**